Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift

```diff
@@ -176,10 +176,9 @@ class TransactionDetailCellBuilder {
                     isOutgoing: isOutgoing,
                     status: tx.txStatus,
                     value: etherTx.value.value,
-                    decimals: eth.decimals.flatMap { try? UInt64($0) },
-                    symbol: eth.symbol,
-                    logoUri: nil,
-                    logo: UIImage(named: "ico-ether"))
+                    decimals: UInt64(coin.decimals),
```

Comment:
decimals field in some places is used as UInt64 and in others is used as UInt256, can we unified them

Reply:
I see, ok

Reply:
will do in a separate pr